types/testingutils/beacon_node.go

```diff
@@ -497,8 +497,12 @@ func (bn *TestingBeaconNode) SubmitAttestation(attestation *phase0.Attestation)
 	return nil
 }
 
+func (bn *TestingBeaconNode) SubmitValidatorRegistration(pubkey []byte, feeRecipient bellatrix.ExecutionAddress, sig phase0.BLSSignature) error {
+	return nil
+}
+
```

Comment:
Shouldn't we have a similar treatment that we do for attestations for tests? 
See `BroadcastedRoots` above

Reply:
done